NitroxServer/ConsoleCommands/StopCommand.cs

```diff
@@ -6,7 +6,7 @@ namespace NitroxServer.ConsoleCommands
 {
     internal class StopCommand : Command
     {
-        public override IEnumerable<string> Aliases { get; } = new[] { "exit", "halt", "quit" };
+        public override IEnumerable<string> Aliases { get; } = new[] { "exit", "halt", "quit", "close" };
```

Comment:
?

Reply:
That change probably isn't in the scope of this PR, but it was a keyword that I thought would be a good addition to the aliases of the stop command (I used it once accidentally before I thought to add it)